code/src/components/NewThoughts.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const NewThoughts = ({ newThought, handleSubmit, newThoughtChange, input }) => {
+  return (
+    <div className='newthought-container'>
+      <h2 className='newthought-header'>What`s making you happy right now?</h2>
+      <form onSubmit={handleSubmit}>
```

Comment:
Using the Eslint in this case would work beacause it gives indentations for each object that is inside another object.